### PR TITLE
Merge new website to master

### PR DESCRIPTION
**Pull request by KiaanCastillo**
# Changes
- Merge develop into master to get new website up and deployed!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,37 @@
...
+
+# testing
+/coverage
+
+# next.js
+/.next/
+/out/
+
+# production
+/build
```
Comment (kylesaburao) [inline .gitignore:17]: This may affect current deployment procedures because build will not be captured by git

Comment (kylesaburao) [reply .gitignore:17]: If build is being updated with the intent for deploy, the deployment isn't going to get updated

Comment (KiaanCastillo) [reply .gitignore:17]: thanks, Ill remove it just in case

Comment (KiaanCastillo) [reply .gitignore:17]: I will not be using a flow that assumes build folder but Ill remove for now

Comment (kylesaburao) [reply .gitignore:17]: also delete build from your computer, commit, rebuild, commit

**Review by kylesaburao — APPROVED** [review]
LGTM

**Pull request merged**